libs/libcds/tests/performance/libcds_hazard_pointer_overhead.cpp

```diff
@@ -287,6 +264,18 @@ int hpx_main(variables_map& vm)
     libcds_wrapper wrapper;
 
     {
+        // Initialize Hazard Pointer singleton
+        const std::size_t nHazardPtrCount =
+            1;    // Hazard pointer count per thread
+        const std::size_t nMaxThreadCount = hpx::cds::hpxthread_manager_wrapper::
+            max_concurrent_attach_thread_;    // Max count of simultaneous working thread in the application
+        const std::size_t nMaxRetiredPtrCount =
+            16;    // Capacity of the array of retired objects for the thread
```

Comment:
Could we stick to HPX naming conventions, please?

Reply:
What is the style of HPX naming conventions?